Documentation/contributing.rst

```diff
@@ -653,6 +724,62 @@ saved, in case of a failing test an exhaustive data will be added.
 	Allocated IPv4 addresses:
 
 
+Running with delve
+^^^^^^^^^^^^^^^^^^
+
+`Delve <https://github.com/derekparker/delve>`_ is a debugging tool for Go
+applications. If you want to run your test with delve,  you should add a new
+breakpoint using
+`runtime.BreakPoint() <https://golang.org/pkg/runtime/#Breakpoint>`_ in the
+code, and run ginkgo using ``dlv``.
+
+Example how to run ginkgo using ``dlv``:
+
+::
+
+	dlv test . -- --ginkgo.focus="Runtime" -ginkgo.v=true --cilium.provision=false
+
+
+Running test in other VMs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to run test in a different VM, you can use ``--cilium.SSHConfig`` to
+provide the SSH configuration of the endpoint on which tests will be ran. The
+tests presume the following on the remote instance:
+
+- Cilium source code is located in the directory ``/home/vagrant/go/src/github.com/cilium/cilium/``.
+- Cilium is installed and running.
+
+The ssh connection need to be defined as a ``ssh-config`` file and need to have
+the following targets:
+
+- runtime: To run runtime tests
+- k8s{1..2}-${K8S_VERSION}: to run Kubernetes tests. Kubernetes tests. These
```

Comment:
> to run Kubernetes tests. Kubernetes tests.

remove " Kubernetes tests" . Please be more careful with proofreading your changes.